src/coreclr/jit/lclvars.cpp

```diff
@@ -3845,6 +3845,57 @@ var_types LclVarDsc::lvaArgType()
     return type;
 }
 
+//------------------------------------------------------------------------
+// GetRegisterType: Determine register type for this local var.
+//
+// Arguments:
+//    tree - node that uses the local, its type is checked first.
+//
+// Return Value:
+//    TYP_UNDEF if the layout is enregistrable, register type otherwise.
```

Comment:
Isn't it the opposite? That is, return TYP_UNDEF if the layout is *NOT* enregistrable?

Reply:
Thanks, fixed.